docroot/themes/humsci/humsci_basic/src/scss/components/_chosen.scss

```diff
@@ -173,55 +162,136 @@
   //
   // }
 }
-
 // Multi Chosen
-// .chosen-container-multi {
-//   .chosen-choices {
-//
-//   }
-//
-//   .chosen-choices li {
-//     &.search-field {
-//       input[type="text"] {
-//
-//       }
-//     }
-//
-//     &.search-choice {
-//
-//       span {
-//
-//       }
-//
-//       .search-choice-close {
-//
-//         &:hover {
-//
-//         }
-//       }
-//     }
-//
-//     &.search-choice-disabled {
-//
-//     }
-//
-//     &.search-choice-focus {
-//
-//       .search-choice-close {
-//
-//       }
-//     }
-//   }
-//
-//   .chosen-results {
-//
-//   }
-//
-//   .chosen-drop .result-selected {
-//
-//   }
-// }
+.chosen-container-multi {
+  .chosen-choices {
+    display: flex;
+    align-items: center;
+    flex-wrap: wrap;
+    // scss-lint:disable ImportantRule
+    background: none !important;
+    // scss-lint:enable ImportantRule
+    box-shadow: none;
+    font-size: hb-calculate-rems(16px);
+    line-height: 1;
+    height: auto;
+    min-height: $hb-input-min-height;
+    padding: 0 hb-calculate-rems(8px) 0 hb-calculate-rems(8px);
+    @include hb-input-border;
+
+    &:focus-within {
+      @include hb-input-focus;
+    }
+  }
+
+  .chosen-choices li {
+    &.search-field {
+      input[type="text"] {
+        // scss-lint:disable ImportantRule
+        width: 0 !important;
+        // scss-lint:enable ImportantRule
+        margin: 0;
+        padding: 0;
+        font-size: hb-calculate-rems(16px);
+        font-family: inherit;
+      }
+    }
+
+    &.search-choice {
+      margin: hb-calculate-rems(5px) hb-calculate-rems(5px) hb-calculate-rems(5px) 0;
+      @include hb-global-color('background-color', 'white');
+      font-family: inherit;
+      background-image: none;
+      box-shadow: none;
+      @include hb-global-color('color', 'gray-dark');
+      border-radius: hb-calculate-rems($hb-border-radius);
+      padding: hb-calculate-rems(6px) hb-calculate-rems(22px) hb-calculate-rems(6px) hb-calculate-rems(6px);
+      transition: 100ms ease-in-out;
+
+      &:hover {
+        @include hb-global-color('background-color', 'gray-light');
+        @include hb-global-color('color', 'black');
+      }
 
+      // when you click on the close button
+      &:focus {
+        @include hb-input-focus;
+      }
+
+      // span {}
+
+      .search-choice-close {
+        // scss-lint:disable ImportantRule
```

Comment:
You don't have to wrap this around each individual rule. You can nest multiple styles in one:
```
// scss-lint:disable ImportantRule
background: none !important;
background-size: 0 !important;
background-repeat: no-repeat: !important;
// scss-lint:enable ImportantRule
```